### DISCUSSION TITLE
pedigree individuals from different populations

### DISCUSSION THREAD

**Discussion by lakishadavid** [tskit-dev/msprime#2149] [Q&A]
I'm using the fixed pedigree with individuals from multiple populations. I want to simulate data where the individuals in generation 0 are AFR or ADMIX from the AmericanAdmixture_4B11 model. When building the pedigree, do I need to create a separate sub-pedigree for each population as shown in [Pedigrees and demography](https://tskit.dev/msprime/docs/latest/ancestry.html#pedigrees-and-demography)? 

**What I did:** I completed the pedigree-aware simulation with multiple populations, but I built the fully connected pedigree without creating sub-pedigrees for each population. If I built a 10-generation pedigree (backwards 0 - 9), the pedigree founders in generation 9 were AFR, leaf founders on the left were AFR for generations 1 - 8 (with their generation 0 = AFR), and leaf founders on the right for generations 1 - 8 were randomly ADMIX or EUR (with their generation 0 = ADMIX).

I'm seeing from [Pedigrees and demography](https://tskit.dev/msprime/docs/latest/ancestry.html#pedigrees-and-demography) that I should have built the pedigree differently.

**Should I?** (1) Create a sub-pedigree with just ADMIX from generation 0 to 8 with founders = ADMIX and no leaf founders, and (2) create a separate sub-pedigree with just AFR from generation 0 to 8 with founders = AFR and no leaf founders, (3) for one pedigree object? (4) And then run the initial and completed simulations?

**Comment by jeromekelleher** [comment]
I think you probably want to simulate the pedigrees for AFR and ADMIX separately, and then join the pedigrees together afterwards (like in the example you link to). If your population labelling is consistent in the demographic model if *should* all work out, in that the simulation will pick up that the genetic material is in the appropriate populations, and then the standard backwards simulation can take over.
